js/open-photo.js

```diff
@@ -0,0 +1,59 @@
+import { photos } from './mock.js';
+import { isEscapeKey, findElement } from './util.js';
+
+const picturesBlock = document.querySelector('.pictures');
+const bigPicture = document.querySelector('.big-picture');
+const closeButton = document.querySelector('.big-picture__cancel');
+const commentsBlock = document.querySelector('.social__comments');
+const commentTemplate = document.querySelector('#comment').content.querySelector('.social__comment');
+
+const closePhoto = () => {
+	bigPicture.classList.add('hidden');
+	document.body.classList.remove('modal-open');
+	document.querySelector('.social__comments').textContent = '';
+};
+
+const buttonClickHandler = () => {
+	closePhoto();
+};
```

Comment:
В принципе мы можем один раз навесить обработчик вне функции открытия и не снимать его, потому что элемент всегда на странице
```suggestion
const buttonClickHandler = () => {
	closePhoto();
};

picturesBlock.addEventListener('click',  buttonClickHandler);
```
Или даже не создавать отдельную функцию, а сделать обёртку чтобы не нарушать тот глупый критерий по названия
```suggestion
picturesBlock.addEventListener('click', () => closePhoto());
```

Reply:
так что по итогу? вынести обработчики клика и нажатия из функции открытия в соответствии с критерием Д28?
и разве `picturesBlock.addEventListener('click', () => closePhoto());` не противоречит Д24? ведь вызываем мы функцию, которая названа не по критерию

Reply:
>  ведь вызываем мы функцию, которая названа не по критерию

Нет, мы сделали безымянную одноразовую функцию. К такому не придираются. На всякий случай проясню:
`closePhoto` и `() => closePhoto()` — это две разные функции. 

Ты можешь выбрать любой из этих вариантов